backend/game/engine/player_queue.py

```diff
@@ -0,0 +1,36 @@
+import itertools
+
+
+class GamePlayers():
+
+    def __init__(self):
+        self.all_players = self.dead_players = self.alive_players = []
+        self.current_player = None
+
+    def _add_initial_player(self, player):
+        self.all_players.append(player)
+
+    def _remove_initial_player(self, player):
```

Comment:
What does this mean? Remove initial player?

Reply:
Depending on the mechanics of the game lobby, didn't know if players were going to be able to queue in/out so that they could be added to the list and removed but then once a 'start game' (or in this case 'set player order') was established, they would be locked in

Reply:
This really depends on how we have the lobby set up. We could have it such that there's like a chatroom-esque pre-game lobby deal where players can figure out if they want to play. Once they all pick their monsters and what not, then they could be redirected to the actual game playing display where they are assigned a playing order and spots.